extensions/elytron-security-jdbc/deployment/src/main/java/io/quarkus/elytron/security/jdbc/deployment/ElytronSecurityJdbcProcessor.java

```diff
@@ -50,7 +50,7 @@ FeatureBuildItem feature() {
     void configureJdbcRealmAuthConfig(JdbcRecorder recorder,
             BuildProducer<SecurityRealmBuildItem> securityRealm,
             BeanContainerBuildItem beanContainerBuildItem, //we need this to make sure ArC is initialized
-            Optional<DataSourceInitializedBuildItem> dataSourceInitialized) throws Exception {
+            List<JdbcDataSourceBuildItem> dataSourcesConfigured) throws Exception {
```

Comment:
This also requires the Javadoc to be updated

Reply:
Done.